Microsoft.Packaging.Tools/tasks/Trimming/TrimFiles.cs

```diff
@@ -55,7 +55,6 @@ public partial class TrimFiles : Task
         /// <summary>
         /// RuntimeIdentifier to use when determining package dependencies.
         /// </summary>
-        [Required]
         public string RuntimeIdentifier { get; set; }
```

Comment:
Yes, I tested this.  The only place we don't explicitly check it for null is when we pass to NuGet.ProjectModel (lockFile.GetTarget), which handles a null value correctly.